srtcore/core.cpp

```diff
@@ -4899,22 +5047,37 @@ int64_t CUDT::sendfile(fstream& ifs, int64_t& offset, int64_t size, int block)
       m_iReXmitCount = 1;
    }
 
-   int64_t tosend = size;
-   int unitsize;
-
    // positioning...
    try
    {
-      ifs.seekg((streamoff)offset);
+       if (size == -1)
```

Comment:
ident != 3 spaces

Reply:
Whole functions have been reformatted to 4-spaced indents. We can discuss total reformatting of the code as a separate activity; the 3-spaced indents come from the original UDT and it's a little bit clumsy. No one really does that in the OS world and 4-spaced indents are rather generally preferred.